assets/schema/ensemble_schema.json

```diff
@@ -2059,4 +2060,4 @@
 
 
   }
-}
\ No newline at end of file
+}
```

Comment:
<img width="848" alt="Screen Shot 2022-09-10 at 8 46 52 AM" src="https://user-images.githubusercontent.com/622269/189491092-cc8cf365-a8c0-472c-9013-e6178f0ccfab.png">

IF you just want to test the schema locally, just reference a local schema file like this